src/build_helper/src/git.rs

```diff
@@ -9,6 +9,16 @@ pub struct GitConfig<'a> {
     pub git_merge_commit_email: &'a str,
 }
 
+impl GitConfig<'_> {
```

Comment:
Nit: could you move this to a free function, such as `cli_escape_author_email`? To make it clear from the name what it does.